front/pages/w/[wId]/subscription/index.tsx

```diff
@@ -276,6 +276,7 @@ export default function Subscription({
                   <Page.H variant="h5">Manage my plan</Page.H>
                   <div className="s-h-full s-w-full pt-2">
                     <PricePlans
+                      display="landing"
```

Comment:
The pricing table is being removed from the settings -> subscription page in an upcoming PR, so no big deal about this line.